beamer/chain.py

```diff
@@ -198,10 +198,12 @@ def _process_requests(self) -> None:
                 claim_request(request, self._context)
 
             elif request.is_withdrawn:
-                active_claims = any([claim.is_withdrawn for claim in self._context.claims])
+                active_claims = any(
+                    claim.request_id == request.id for claim in self._context.claims
```

Comment:
So,m when a claim is withdrawn, it will also be removed: https://github.com/beamer-bridge/beamer/pull/515/files#diff-59108d402087b9f4bfab36ec32d92fd39d9c477b17ac359eeaaece1104e0f699L225

So based on this, we can just say here: Whenever there is a claim that belongs to this request, we should just wait until the claims are handled and removed.

Reply:
Thanks for explaining. 👌🏾